GraalVM/CE/1.0.0-rc14/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2015 Oracle and/or its affiliates. All rights reserved.
+#
+
+FROM oraclelinux:7-slim
+
+# Note: If you are behind a web proxy, set the build variables for the build:
+#       E.g.:  docker build --build-arg "https_proxy=..." --build-arg "http_proxy=..." --build-arg "no_proxy=..." ...
+
+ARG GRAAL_VERSION=1.0.0-rc14
+ENV LANG=en_US.UTF-8
+
+ENV GRAALVM_PKG=https://github.com/oracle/graal/releases/download/vm-$GRAAL_VERSION/graalvm-ce-$GRAAL_VERSION-linux-amd64.tar.gz \
+    JAVA_HOME=/opt/graalvm-ce-$GRAAL_VERSION/ \
+    PATH=$PATH:/opt/rh/llvm-toolset-7/root/usr/bin \
+    LD_LIBRARY_PATH=/opt/rh/llvm-toolset-7/root/usr/lib64 \
+    MANPATH=/opt/rh/llvm-toolset-7/root/usr/share/man \
+    PKG_CONFIG_PATH=/opt/rh/llvm-toolset-7/root/usr/lib64/pkgconfig \
+    PYTHONPATH=/opt/rh/llvm-toolset-7/root/usr/lib/python2.7/site-packages \
+    X_SCLS=llvm-toolset-7
+
+RUN curl -o /etc/yum.repos.d/public-yum-ol7.repo http://yum.oracle.com/public-yum-ol7.repo \
```

Comment:
We no longer support the monolithic yum repo file. Please update to using the individual release packages as documented here: https://yum.oracle.com/getting-started.html#checking-yum-configuration

In this instance, you'll want to do a `yum update oraclelinux-release-el7` and a `yum install oraclelinux-developer-release-el7  oracle-softwarecollection-release-el7`

Reply:
Thanks, this looks better.